### DISCUSSION TITLE
Is it possible to select folder(s) in the gallery and return the path(s)?

### DISCUSSION THREAD

**Discussion by strangeways-dev** [yasirkula/UnityNativeGallery#276] [Q&A] [answered]
Is it possible to select a folder / folders in the gallery and return the path(s)?
I have a feature on my app that allows a user on windows etc, to be able to pick a folder and it imports all the images from that folder and takes the name of the folder and uses it to group the imported images.

I can make that possible on android but I need to be able to get the path of a folder.
Cheers.

**Comment by yasirkula** [comment, marked as answer]
This is unfortunately not possible with NativeGallery 😞
